### PR TITLE
add asgi support to docker

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

**Comment by annashamray** [comment]
**Changes:**
1. Replace wsg` server (uwsgi) with the asgi one `(daphne`)
1. Add `nginx` to docker configuration to manage static files

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,8 @@ WORKDIR /app
 
 COPY ./requirements.txt /app/
 RUN /app/env/bin/pip install -r requirements.txt
-RUN /app/env/bin/pip install uwsgi
+#RUN /app/env/bin/pip install uwsgi
+RUN /app/env/bin/pip install daphne
```
Comment (sergei-maertens) [inline Dockerfile]: add this to `requirements.in` please, always use pinned versions

File: bin/docker_start.sh [hunk 2]
```diff
@@ -24,12 +24,15 @@ python src/manage.py migrate
...
-    --static-map /static=/app/static \
-    --static-map /media=/app/media  \
-    --chdir src \
-    --processes 2 \
-    --threads 2
-    # processes & threads are needed for concurrency without nginx sitting inbetween
+cd src
+daphne -p $asgi_port -b 0.0.0.0 zac.asgi:application
+
+#uwsgi \
```
Comment (sergei-maertens) [inline bin/docker_start.sh]: remove code instead of commenting please

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
https://github.com/VNG-Realisatie/gemma-zaken-demo/blob/master/src/zac/conf/settings.py#L195 - this is not going to work, the redis host will have to be passed as envvar as well so that it can talk to a redis DB in a different container.

File: Dockerfile (hunk 1 above)
Comment (annashamray) [reply Dockerfile]: `requirements.in` is updated

File: bin/docker_start.sh (hunk 2 above)
Comment (annashamray) [reply bin/docker_start.sh]: commented code is removed

**Review by annashamray — COMMENTED** [review]
I explicitly added `REDIS_HOST` to `docker-compose.yml`, but it could work without it because variable REDIS_HOST is assigned in `conf/docker.py`

File: Dockerfile (hunk 1 above)
Comment (sergei-maertens) [reply Dockerfile]: I've removed this line because now it's installed through `requirements.txt`

**Comment by sergei-maertens** [comment]
I missed that the `REDIS_HOST` was defined in `docker.py` settings file. The settings are a mess, I'll condense them down to minimal set of overrides.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**